applications/asset_tracker_v2/src/modules/Kconfig.debug_module

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DEBUG_MODULE
+	bool "Debug module"
+
```

Comment:
I think we need a guard here somewhere, otherwise all these options will show up in menuconfig and other UIs also when the debug module is disabled